packages/react-scripts/template/README.md

```diff
@@ -1972,6 +1980,44 @@ npm run build
 npm run analyze
 ```
 
+### Using Webpack Stats JSON
+
+Webpack can produce a JSON manifest that details the bundles, and several tools can use that file to do analysis.
+
+Unlike with sourcemaps, the JSON file isn't created automatically on build. You must pass a `--stats` flag, using `--` to ensure it's passed to `react-scripts`:
```

Comment:
The second sentence in this paragraph feels like it's too "inside baseball"; like we shouldn't need to mention what's happening with `react-scripts` under the hood.

But if I omit it, I worry that we aren't explaining the `--`, and this might cause folks to not notice it. I can imagine people being frustrated when `npm run build --stats` doesn't work and they can't figure out why?

Maybe there's a happy middle-ground. Open to copy suggestions.

Reply:
This seems like a nice, succinct explanation to me. I don't have trouble with mentioning `react-scripts` here, maybe @gaearon has ideas?

Reply:
I don't think `--` is needed with Yarn actually, maybe it's worth to mention...

Reply:
True. With yarn, we actually get a deprecation warning when using `--`. It might be a good idea to note it.

Reply:
What's the process to merge this PR? From what I see the only required changes are on the README.md file.

Reply:
I think everything after the word `flag` can be removed. There's an example of the full command on the next line.